src/ensembl/src/shared/types/thoas/metadata.ts

```diff
@@ -14,13 +14,16 @@
  * limitations under the License.
  */
 
-import { Source } from './source';
+type ValueSetMetadata = {
```

Comment:
according to thoas docs, there is also a description field. Do you not need it? As discussed i have created my branch for the transcript more info from this branch and i am going to add it to the code unless its not needed.

Reply:
> Do you not need it?

It isn't populated with any real data at the moment, and we aren't using it in the client anyway. Currently, the definition field is sufficient for us.